CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+# 8.0.2
+- Fix default logging level to `GULLoggerLevelNotice`; previously unset. (#207)
```

Comment:
Please incorporate the regression was introduced in 8.0.0.

Perhaps: `Fix default logging level regression introduce in 8.0.0`

Reply:
Good idea. Does this seem alright now?

Reply:
Yep. Thanks!